e2e/test/scenarios/embedding/interactive-embedding.cy.spec.js

```diff
@@ -891,13 +891,62 @@ describe("scenarios > embedding > full app", () => {
         () => {
           H.restore("postgres-12");
           cy.signInAsAdmin();
+          H.createModelFromTableName({
```

Comment:
I created a new model, so it's obvious in the test that when joining, only some models should be visible.